### ISSUE TITLE
next-line jumps more than 1 page after doing isearch-forward with smooth-scrolling being enabled

### ISSUE THREAD

**Issue by tddsg** [syl20bnr/spacemacs#6934]
#### Description
The `next-line` command jumps more than 1 page after doing `isearch-forward` when `smooth-scrolling` being enabled.

#### Reproduction guide
- Start Emacs
- Enable the `smooth-scrolling` mode,
- Do `isearch-forward` so that the searched text appearing near the end of the current window. For example, when searching for the "holy" word, and this word is at the last line of the window.
 
![screenshot from 2016-08-27 19 07 11](https://cloud.githubusercontent.com/assets/20593743/18026994/e417ec48-6c89-11e6-92aa-ad4974bb11db.png)

- Now, press `C-n` for the down arrow key to call `next-line`, the cursor will not jump to the next line, but it will jump to the next page instead.

*Observed behaviour:*

After searching by `isearch-forward` and call `next-line`, the cursor jumps to next page.

*Expected behaviour:*

Since my current edging area for `smooth-scrolling` is 5 lines from the bottom and the top, so I don't know what should be done here if the searched text appear inside the edging area.

However, it is expected that `next-line` should jump to the next line...

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.22
- Spacemacs branch: master (rev. 9f9faa4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
(auto-completion better-defaults emacs-lisp git markdown org
                 (shell :variables shell-default-height 30 shell-default-position 'bottom)
                 spell-checking syntax-checking version-control semantic latex ocaml haskell tddsg)
```

#### Backtrace
```
BACKTRACE IF RELEVANT
```

**Comment by TheBB** [comment]
This is fixed in develop.

**Comment by tddsg** [comment]
Thanks for the update! Is the `develop` branch stable to be used? I switched to this branch and encountered some error while loading packages.

**Comment by TheBB** [comment]
It should be reasonably stable. Of course bugs may occur but they are treated as bugs and not guests. :-)

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
